test-suite/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,10 @@
+microstream:
+  storage:
+    main:
+      storage-directory: build/microstream
+      channel-count: 4
+    secondary:
+      channel-count: 4
+      channel-directory-prefix: 'channel_'
+      data-file-prefix: 'channel_'
+      data-file-suffix: '.dat'
```

Comment:
Adding this is causing the tests to fail...

I'm not sure what `secondary` was for, but to get the tests passing, we need to change the constructor of `CustomerRepositoryImpl` to

```
    CustomerRepositoryImpl(@Named("main") EmbeddedStorageManager embeddedStorageManager) { // <1>
```

In both the Java and Groovy test-suites

Does secondary have a special meaning here?  Or is it there for future work?

Reply:
`secondary`will become the name qualifier.

Reply:
Cool, so we just need to add `Named` to the test classes that inject EmbeddedStorageManager (as above)?